src/greptimeai/patchee/openai_patchee/audio.py

```diff
@@ -13,23 +13,26 @@ def __init__(self, client: Union[OpenAI, AsyncOpenAI, None] = None):
         audio_speech_create = Patchee(
             obj=client.audio.speech if client else openai.audio.speech,
             method_name="create",
-            span_name="audio.speech.create",
+            span_name="openai_speech",
+            event_name="audio.speech.create",
         )
 
         audio_raw_speech_create = Patchee(
             obj=client.audio.with_raw_response.speech
             if client
             else openai.audio.with_raw_response.speech,
             method_name="create",
-            span_name="audio.with_raw_response.speech.create",
+            span_name="openai_speech",
+            event_name="audio.with_raw_response.speech.create",
```

Comment:
I find these `event_name`s are a little bit confusing

- audio.with_raw_response.speech.create
- audio.speech.with_raw_response.create
- with_raw_response.audio.speech.create

Is this designed or following some upstream convention?

Reply:
this is the call chain in OpenAI, such as [accessing-raw-response](https://github.com/openai/openai-python#accessing-raw-response-data-eg-headers)